File: idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/RemoteServerConfigGUI.java
package com.itangcent.idea.plugin.dialog;

import javax.swing.*;

public class RemoteServerConfigGUI {
    private JPanel rootJPanel;
    private JTextField token;
    private JTextField serverId;

    public RemoteServerConfigGUI() {
    }

    public JPanel getRootJPanel() {
        return rootJPanel;
    }

    public void setRootJPanel(JPanel rootJPanel) {
        this.rootJPanel = rootJPanel;
    }

    public JTextField getToken() {
        return token;
    }

    public void setToken(JTextField token) {
        this.token = token;
    }

    public JTextField getServerId() {
        return serverId;
    }

    public void setServerId(JTextField serverId) {
        this.serverId = serverId;
    }
}

File: idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/RemoteServerConfigGUI.kt
package com.itangcent.idea.plugin.dialog;

import javax.swing.*;

class RemoteServerConfigGUI {
    private lateinit var rootJPanel: JPanel
    private lateinit var token: JTextField
    private lateinit var serverId: JTextField
    private lateinit var domain:JTextField

    fun getRootPanel(): JComponent {
        return rootJPanel
    }

    fun getToken(): JTextField {
        return token
    }

    fun getServerId(): JTextField {
        return serverId
    }

    fun getDomain():JTextField{
        return domain
    }
}
